.ci/validation.sh

```diff
@@ -262,6 +262,22 @@ verify-no-exception-configs)
   exit $fail
   ;;
 
+verify-regexp-id)
+  cd config
+  RESULT=0
```

Comment:
lets rename it to `fail` as other scripts do

Reply:
done